topics/contributing/tutorials/github-contribution/tutorial.md

```diff
@@ -257,13 +387,84 @@ Once the pull is open, it will be reviewed. There are two possible outcomes:
 1. Your pull request is accepted. Congratulations! Your changes will be merged into the main branch of the original repository. The website will be re-built and you will be in the [Hall of Fame]({% link hall-of-fame.md %})
 2. Your pull request needs modifications: the reviewers will ask for some changes, possibly because the automatic tests are failing.
 
-# Make the requested changes
+# Make any requested changes
+
+<div class="Desktop" markdown="1">
+
+Three things can happen:
+
+1. A reviewer commits changes to your pull request (they fix something themselves)
+2. A reviewer suggests changes to your pull request (they think something will be helpful, but want you to check and decide)
+3. A reviewer posts comments, asking you to make some sort of change.
+
+On larger pull requests, you can end up with all three! Here's how to deal this.
+
+## 1: A reviewer commits their own changes to your pull request
+
+These changes currently only exist in your Github repository online, not on your local branch. In case you have to make any further changes, it's important you *fetch* those changes onto your local branch. This is how you stay up to date.
```

Comment:
```suggestion
These changes currently only exist in your GitHub repository online, not on your local branch. In case you have to make any further changes, it's important you *fetch* those changes onto your local branch. This is how you stay up to date.
```